.github/workflows/autorelease.yml

```diff
@@ -41,8 +42,13 @@ jobs:
     - name: Install Chez Scheme, LLVM & libuv
       run: sudo apt-get install -y chezscheme llvm-15 libuv1-dev
 
-    - name: Run tests
-      run: sbt clean test
+    - name: Run tests with retry
+      uses: nick-fields/retry@v3
+      with:
+        timeout_minutes: 120 # NOTE: This needs _some_ value. As of writing this, 2 hours should be future-proof. :)
+        max_attempts: 3
+        retry_on: error
+        run: sbt clean test
```

Comment:
If we don't want to depend on an external action, we could also add something like the following:
```yaml
    - name: Run tests (Attempt 1)
      id: test_attempt_1
      continue-on-error: true
      run: sbt clean test

    - name: Run tests (Attempt 2)
      id: test_attempt_2
      if: steps.test_attempt_1.outcome == 'failure'
      continue-on-error: true
      run: sbt clean test

    - name: Run tests (Attempt 3)
      id: test_attempt_3
      if: steps.test_attempt_2.outcome == 'failure'
      run: sbt clean test
```